openarena-q3e

```diff
@@ -1,29 +1,66 @@
 #!/bin/sh
-OA_CVARS="+set fs_basegame baseoa +set protocol 71"
-DISABLED_CVARS="+set com_protocol 72 +set cl_voip 1"
 
+###
+CONFIG_FILE=".var/app/io.github.ec_.Quake3e.OpenArena/config/cfg"
+# Check if the config file exists
+if [ -f "$CONFIG_FILE" ]; then
+    source "$CONFIG_FILE"
+fi
+# Check if it's the first run or if RESPECT_XDGDIRS is not set
+if [ ! -f "$CONFIG_FILE" ] || [ -z "$RESPECT_XDGDIRS" ]; then
+    RESPECT_XDGDIRS=0
+
+    # Save RESPECT_XDGDIRS in the config file
+    echo "RESPECT_XDGDIRS=$RESPECT_XDGDIRS" > "$CONFIG_FILE"
+    echo -e "It seems this is the first time you run OpenArena... You can edit the $$CONFIG_FILE and change RESPECT_XDGDIRS to use your XDG_DATA_FOLDER, which is: ${XDG_DATA_HOME}/homepath Instead of polluting your HOMEDIR by adding the hidden .openarena folder. \n"
```

Comment:
Same problem as with the other echo message. This OpenArena Flatpak is not creating the files directly in your home directory.

It is using these directories in your home:
`~/.var/app/io.github.ec_.Quake3e.OpenArena/data/homepath`
`~/.var/app/io.github.ec_.Quake3e.OpenArena/.openarena`

Reply:
```
[anto@beef ~]$ ls -a
.              .bashrc   .local            .xsession-errors.old  Templates
..             .cache    .mozilla          Desktop               Videos
.ICEauthority  .config   .openarena        Documents             amber.icc
.Xauthority    .dbus     .pki              Downloads
.bash_history  .dmrc     .var              Music
.bash_logout   .gnupg    .wget-hsts        Pictures
.bash_profile  .inputrc  .xsession-errors  Public

```